src/modules/project/pages1/projectFunding/views/fundingInit/sections/GeyserTipInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, ButtonGroup, HStack, IconButton, VStack } from '@chakra-ui/react'
+import { t } from 'i18next'
+import { PiQuestion } from 'react-icons/pi'
+
+import { Tooltip } from '@/components/ui/Tooltip.tsx'
+import { useFundingFormAtom } from '@/modules/project/funding/hooks/useFundingFormAtom.ts'
+import { CardLayout } from '@/shared/components/layouts/CardLayout.tsx'
+import { H3 } from '@/shared/components/typography/Heading.tsx'
+
+/** GeyserTipInput component for selecting a tip percentage for Geyser */
+export const GeyserTipInput = () => {
+  const { formState, setGeyserTipPercent } = useFundingFormAtom()
+  const { geyserTipPercent } = formState
+
+  const tipOptions = [0, 5, 10]
+
+  return (
+    <CardLayout w="full" spacing={3}>
+      <VStack align="start" w="full" spacing={3}>
+        <HStack spacing={4} align="center" justify="space-between" w="full">
```

Comment:
Three consecutive flex boxes are rarely necessary, maybe 2 for unique styling reasons. COuld be reduced to 1 or 2.